src/owncloud/apache_config

```diff
@@ -14,3 +14,21 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+# add fpm
+<IfModule proxy_fcgi_module>
+	<Proxy "unix:/var/run/php.socket|fcgi://php-fpm" timeout=300>
+	</Proxy>
+</IfModule>
+
+<Directory "${SNAP}">
+	<IfModule proxy_fcgi_module>
+		<FilesMatch \.php$>
+			SetHandler "proxy:fcgi://php-fpm/"
+		</FilesMatch>
+	</IfModule>
+</Directory>
+
+# this should route all php files though fpm - note that fpm has to run with the same rights as apache (to own the files)
+ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:9000/srv/www/htdocs/owncloud/$1
+DirectoryIndex /index.php index.php
```

Comment:
... but I'm not entirely sure this has to be here. I know it has to end up in the vhost file of ownCloud (at least that's where I put it on my system) but is this going there?